x-pack/plugins/security_solution/public/management/components/console/components/console_state/state_reducer.ts

```diff
@@ -36,6 +38,13 @@ export const stateDataReducer: ConsoleStoreReducer = (state, action) => {
 
     case 'executeCommand':
       return handleExecuteCommand(state, action);
+
+    case 'updateCommandStatusState':
```

Comment:
Out of curiosity, why do we need those actions with the same result?

Reply:
Yes. The handlers should support these (and maybe future) state changes to the Command's `state` property.